component/xload/stats_manager.go

```diff
@@ -160,7 +167,11 @@ func (sm *StatsManager) statsProcessor() {
 
 		case SPLITTER:
 			// log.Debug("statsManager::statsProcessor : splitter: Name %v, success %v, download %v", item.name, item.success, item.download)
-			sm.updateSuccessFailedCtr(item.Success)
+			if item.Download {
+				sm.updateSuccessFailedCtr(item.Success)
+			} else {
+				sm.updateDiskStats(item.BytesTransferred)
+			}
```

Comment:
We should add some boolean field in `StatsItem` to indicate that this stat is for the bytes written to disk.

Reply:
done